applications/plugins/SofaAssimp/SceneColladaLoader.cpp

```diff
@@ -81,13 +81,16 @@ SceneColladaLoader::SceneColladaLoader() : SceneLoader()
   , importer()
   , animationSpeed(initData(&animationSpeed, 1.0f, "animationSpeed", "animation speed"))
   , generateCollisionModels(initData(&generateCollisionModels, true, "generateCollisionModels", "generate point/line/triangle collision models for imported meshes"))
-  #if COLLADASCENELOADER_HAVE_FLEXIBLE
+  , collisionProximity(initData(&collisionProximity, 0.01f, "contactProximity", "collision proximity for the generated collision models"))
+  , collisionStiffness(initData(&collisionStiffness, 10.0f, "contactStiffness", "collision stiffness for the generated collision models"))
+  , collisionFriction(initData(&collisionFriction, 0.0f, "contactFriction", "collision friction for the generated collision models"))
+#if COLLADASCENELOADER_HAVE_FLEXIBLE    
```

Comment:
Note for myself: remove this conditional code. The proper design is to make a child class to add theses.